python/GafferTest/TestCase.py

```diff
@@ -114,11 +114,11 @@ def tearDown( self ) :
 		## \todo Fix Cortex so that wrapped classes don't require garbage collection.
 		IECore.RefCounted.collectGarbage()
 
-		for root, dirs, files in os.walk( self.temporaryDirectory() ) :
-			for fileName in [ p for p in files + dirs if not ( pathlib.Path( root ) / p ).is_symlink() ] :
-				( pathlib.Path( root ) / fileName ).chmod( stat.S_IRWXU )
-
 		if self.__temporaryDirectory is not None :
+			for root, dirs, files in os.walk( self.temporaryDirectory() ) :
```

Comment:
Minor quibble - let's use `__temporaryDirectory` here rather than `temporaryDirectory()`, to match the other uses in this block. We're saying "the directory we made it earlier" rather than "make the directory".

Reply:
Sounds good, fixed in https://github.com/GafferHQ/gaffer/pull/6204/commits/2d73134aec1ebb91afa805b790f89748ec607605